CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-07-16
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240716.
+
+### Announcements
+
+* Starting 1.30 Kubernetes version and 1.27 LTS versions, beta APIs will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta APIs closer to the 1.30 release.
```

Comment:
This release includes the GA of 1.30. Do we have the option for enabling beta APIs released?